src/test/java/cz/muni/pa165/teamwhite/formula1/UserTest.java

```diff
@@ -0,0 +1,102 @@
+package cz.muni.pa165.teamwhite.formula1;
+
+import cz.muni.pa165.teamwhite.formula1.dao.UserDao;
+import cz.muni.pa165.teamwhite.formula1.entity.User;
+import cz.muni.pa165.teamwhite.formula1.enums.Role;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.Rollback;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import javax.persistence.*;
```

Comment:
Do not import everything, please... :cry:

Reply:
solved